generator/internal/scaffold_generator.cc

```diff
@@ -305,25 +304,30 @@ package(default_visibility = ["//visibility:private"])
 
 licenses(["notice"])  # Apache 2.0
 
+service_dirs = [
```

Comment:
Is this the conclusion from https://github.com/googleapis/google-cloud-cpp/pull/10278#discussion_r1027103136?

Reply:
I think the for loops will be easier to maintain, because they are more generic. We maintain one list for each library: `service_dirs`. (And the generator_config knows the values, so I bet we will automate this maintenance one day).

I updated the definitions to be more succinct, as suggested in the comment (sorta). I figured I might as well update `vmwareengine/BUILD.bazel` too.

Reply:
SGTM.

We should remember to update the couple of google3 BUILD files to match after the next import.